high_card.rb

```diff
@@ -1,5 +1,48 @@
+require "pry"
+
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+deck_cards = []
+
+# class Cards
+
+  def deck
+    # It looks like we will need a nested for loop
+    for rank in ranks do #we used an each loop in the hw review; "each is far more ubiquitous in Ruby"
+      # use .each_with_index to pull worth of card
+      for suit in suits do
+        # combine every combination of indexes from ranks and suite arrays
+        deck_cards << "#{rank} of #{suit}"  # the readme says this should be an array of arrays
```

Comment:
It would benefit you to pass in something other than a string so that you can easily extract `rank` and `suit` later on in your application. I would recommend trying out a hash.